src/components/Bar.svelte

```diff
@@ -4,7 +4,7 @@
 </button>
 {{/if}}
 
-<h1 class='{{titleClass}}'>
+<h1 data-lang='{{lang}}' class='{{titleClass}}'>
```

Comment:
Why `data-lang` ? You can also use `lang` directly (is useful for screen readers).